tests/v2/validation/Jenkinsfile_pre_post_upgrade

```diff
@@ -0,0 +1,151 @@
+#!groovy
```

Comment:
I think I've asked about this before to you @igomez06, both for Natalie's and my task, are we going to push the validation Jenkinsfiles in `validation/pipeline` as a separate dir? What do you guys think?

Reply:
I'm not sure, but my instinct is that leaving all the jenkinsfiles in `validation` makes it a bit more obvious what's going on to someone who's brand new to the project/unfamiliar with Jenkins, at the cost of a small amount of clutter in `validation`, which seems fine for now.

Reply:
I think for now this is fine Ugur, when we start adding more Jenkins files we can readdress this.